log_outgoing_requests/models.py

```diff
@@ -16,6 +18,18 @@
 logger = logging.getLogger(__name__)
 
 
+class OutgoingRequestsLogQueryset(models.QuerySet):
+    def cleanup_requests_logs(self):
```

Comment:
```suggestion
    def prune(self):
```
I think this reads a bit more naturally: `OutgoingRequestsLog.objects.prune()`